### PR TITLE
Excel service

### PR DESCRIPTION
**Pull request by SBriscoe2**
Amended Model, Resource file, Controller, Service and Views.

### CODE REVIEW

File: workforceDashboard/src/main/java/gov/homeoffice/workforceDashboard/service/EmployeeServiceImpl.java [hunk 1]
```diff
@@ -24,4 +30,112 @@ public void saveEmployee (Employee employee){
...
+                employee.setTargetResourceType(targetResourceType);
+                employee.setProjectedStartDateOfRole(projectedStartDateOfRole);
+                employee.setProjectedEndDateOfTerminatingRole(projectedEndDateOfTerminatingRole);
+                employee.setfTE(fTE);
+                employee.setdDaTOrNonDDaTResource(dDaTOrNonDDaTResource);
+
+                System.out.println("Employee record saved--" + employee.toString());
+                saveEmployee(employee);
+
+            } file.close();
```
Comment (catherine-nakka) [inline workforceDashboard/src/main/java/gov/homeoffice/workforceDashboard/service/EmployeeServiceImpl.java]: We should be closing the workbook as well after processing it

File: workforceDashboard/src/main/java/gov/homeoffice/workforceDashboard/service/EmployeeServiceImpl.java [hunk 2]
```diff
@@ -24,4 +30,112 @@ public void saveEmployee (Employee employee){
...
+                String operationalLineManagerSurname = row.getCell(24).getStringCellValue();
+                String operationalLineManagerRoleReference = row.getCell(25).getStringCellValue();
+                String currentResourceType = row.getCell(26).getStringCellValue();
+                String targetResourceType = row.getCell(27).getStringCellValue();
+                String projectedStartDateOfRole = row.getCell(28).getStringCellValue();
+                String projectedEndDateOfTerminatingRole = row.getCell(29).getStringCellValue();
+                Double fTE = row.getCell(30).getNumericCellValue();
+                String dDaTOrNonDDaTResource = row.getCell(31).getStringCellValue();
+
+                employee.setRoleRef(roleRef);
```
Comment (catherine-nakka) [inline workforceDashboard/src/main/java/gov/homeoffice/workforceDashboard/service/EmployeeServiceImpl.java]: We can half the number of lines here by assigning the cell value directly to the setter method. Saves us from declaring all of them first and referring to the same value twice.

File: workforceDashboard/src/main/java/gov/homeoffice/workforceDashboard/service/EmployeeServiceImpl.java (hunk 1 above)
Comment (SBriscoe2) [reply workforceDashboard/src/main/java/gov/homeoffice/workforceDashboard/service/EmployeeServiceImpl.java]: Completed.  See latest commit

File: workforceDashboard/src/main/java/gov/homeoffice/workforceDashboard/service/EmployeeServiceImpl.java (hunk 2 above)
Comment (SBriscoe2) [reply workforceDashboard/src/main/java/gov/homeoffice/workforceDashboard/service/EmployeeServiceImpl.java]: Completed see latest commit

**Pull request merged**